packages/waku/src/lib/utils/stream.ts

```diff
@@ -31,6 +56,13 @@ export const streamToString = async (
   return outs.join('');
 };
 
+export function arrayBufferToString(buffer: ArrayBuffer): string {
+  const uint8Array = new Uint8Array(buffer);
+  return Array.from(uint8Array)
```

Comment:
I'm not familiar but does it create a new array?

Reply:
array.from does yes, I think we should try to avoid conversions if possible, or convert once and handle from there. this feels like a hack to me